controller/src/beerocks/bml/CMakeLists.txt

```diff
@@ -38,16 +38,7 @@ target_include_directories(${PROJECT_NAME} PRIVATE
 
 # Install
 install(
```

Comment:
I can't comment on the commit message, but you are right - bml must be exported, this is the interface to the controller. So all the removed exports which are needed need to come back.

Reply:
I propose to first remove it all, and then add it again later in a consistent way. Does that work for you?

Reply:
Works fine as long as we keep it separate (not merge it yet), since I want to sync our internal repos to what we did so far and this will break it, what do you think?